.sage/main.go

```diff
@@ -51,6 +52,22 @@ func GoLint(ctx context.Context) error {
 	return sggolangcilint.Run(ctx)
 }
 
+func GoFix(ctx context.Context) error {
+	sg.Logger(ctx).Println("fixing Go files...")
+	return sggolangcilint.Fix(ctx)
+}
+
+func GoLines(ctx context.Context) error {
+	sg.Logger(ctx).Println("wrapping Go lines...")
+	return sggolines.Run(ctx)
+}
+
+func GoFormat(ctx context.Context) error {
+	sg.Logger(ctx).Println("formatting Go files...")
+	sg.SerialDeps(ctx, GoFix, GoLines)
+	return nil
+}
```

Comment:
What about gofmt?

Reply:
I'll let @odsod answer but isn't `gofumpt` that, and more?

Reply:
Ah didn't realize the golangci-lint fix target was running gofumpt. Yeah I think it covers gofmt as well